ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp

```diff
@@ -1233,6 +1234,60 @@ void bind_unary_operation_with_diag(py::module& module, const unary_operation_t&
 }
 
 
+template <typename unary_operation_t>
+void bind_dropout(py::module& module, const unary_operation_t& operation) {
+    auto doc = fmt::format(
+        R"doc({0}(input_tensor: ttnn.Tensor, *, seed: uint32_t, probability: float, scale: float, memory_config: Optional[ttnn.MemoryConfig] = None) -> ttnn.Tensor
+
+            Applies {0} to :attr:`input_tensor` element-wise.
+
+            .. math::
+                {0}(\\mathrm{{input\\_tensor}}_i)
+
+            Args:
+                * :attr:`input_tensor`
+
+            Keyword Args:
+                * :attr:`seed` (float): seed used for RNG
```

Comment:
Why float, but if it is float then what about line 1240 ?

Reply:
Good catch, will fix it.